devnet/src/mpc.rs

```diff
@@ -387,7 +397,22 @@ struct JoinArgs {
     sign_pk: near_sdk::PublicKey,
 }
 
-impl MpcVoteJoinCmd {
+/// Gets a list of voters who would send the vote txn, based on the cmdline flag (empty list means
+/// all participants; otherwise it's the precise list of participant indices).
+fn get_voter_account_ids<'a>(
+    mpc_setup: &'a MpcNetworkSetup,
+    voters: &[usize],
+) -> Vec<&'a AccountId> {
```

Comment:
Maybe just `get_network_account_ids()` and `node_ids: &[usize]` instead of `voters`?

The term "voter' implies that the corresponding AccountId is a registered participant in the contract, but we don't have that guarantee here. Afai understood, `MpcNetworSetup` just contains all nodes that belong to our cluster and they might or might not all be participants.

Reply:
Well.... without the specific context then the special case of "node_ids" being empty would be weird :)

Reply:
I think I missed something. Isn't it that mpc_setup contains all mpc nodes of our cluster, but not all of them are participants? E.g. we could deploy a cluster of 5 nodes and then kick 2 of them out during a resharing.

Reply:
Yes, but the vector that is being passed in is supposed to come from the CLI. The behavior is:
 if you do something like `./devnet vote-new-parameters`, this will vote from all accounts
but if you do something like `./devnet vote-new-parameters --voters 0`, it will only vote from the 0th account.
So there's the discontinuity when the voters vector is empty, which is what the default value will be.